Sources/GRPC/ClientConnection.swift

```diff
@@ -325,21 +326,33 @@ extension ClientConnection {
   ///
   /// - Parameter configuration: The configuration to prepare the bootstrap with.
   /// - Parameter group: The `EventLoopGroup` to use for the bootstrap.
-  /// - Parameter timeout: The connection timeout in seconds. 
+  /// - Parameter timeout: The connection timeout in seconds.
   private class func makeBootstrap(
     configuration: Configuration,
     group: EventLoopGroup,
     timeout: TimeInterval?,
     logger: Logger
   ) -> ClientBootstrapProtocol {
+    // Provide a server hostname if we're using TLS. Prefer the override.
+    let serverHostname: String? = configuration.tls.map {
+      if let override = $0.hostnameOverride {
```

Comment:
nit: instead of shadowing the `override` keyword here does it make sense to call this hostnameOverride (it's legal in this context but GitHub's syntax coloring is confused)?

Reply:
Sure, it's a better name anyway!